jsx/Form.js

```diff
@@ -23,7 +23,7 @@ CheckboxElement, ButtonElement, LorisElement
  *
  */
 
-import React, {Component} from 'react';
+import React, {useState, Component} from 'react';
```

Comment:
not in **24.1**

_this is a pretty significant change if I decided to start including `useState` in Form.js. I will look for the ramifications of this in the following changes._

to adopt **24.1** if easily achievable — if not, include as **CBIG override** and remove before merging with **main**